composer.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "keboola/storage-driver-big-query",
+    "description": "Keboola Big Query driver",
```

Comment:
mel bych pripominku k nazvu - BigQuery bych nechal vsude jako jedno slovo, takze:
- repo `storage-driver-bigquery`
- name: `keboola/storage-driver-bigquery`
- desc: Keboola BigQuery driver

Taky bys nerozdelil `MySql` na `driver-my-sql` ;-)

Reply:
Jo to je vlastne pravda dik upravim to :)

Reply:
Upravene